sormas-ui/src/main/java/de/symeda/sormas/ui/caze/MergeCasesView.java

```diff
@@ -97,7 +97,7 @@ public MergeCasesView() {
 
 		addHeaderComponent(btnBack);
 
-		if (viewConfiguration.isFiltersApplied()) {
+		if (!viewConfiguration.isFiltersApplied()) {
```

Comment:
the is the same condition as in the `enter` method but it does the opposite si it is confusing. Sometimes we reload when the filters are not applied and sometimes we show a message to the user, it should be consistent.

Also by default this flag is false so it loads and shows the user a message the he should click on the Apply filters button to see the duplicates

Reply:
update the change